process_pr.py

```diff
@@ -1049,7 +1093,8 @@ def process_pr(repo_config, gh, repo, issue, dryRun, cmsbuild_user=None, force=F
             if pr.state != "closed":
                 logger.error("This pull request must go in to master branch")
                 if not dryRun:
-                    edit_pr(repo.full_name, prId, base="master")
+                    # edit_pr(repo.full_name, prId, base="master")
```

Comment:
@iarspider , so we can remove the import of `edit_pr` ... right? 
also remove the commented line